File: student.py
class Student:
    def __init__(self, name: str, years_old: int, grade: str):
        self.name = name
        self.years_old = years_old
        self.grade = grade

    def get_info(self) -> str:
        return f"Name: {self.name}, Age: {self.years_old}, Grade: {self.grade}"

    def is_adult(self) -> bool:
        return self.years_old >= 18

File: student.ts
export function calculateSum(a: number, b: number): number {
  return a + b;
}

export class Student {
  constructor(
    public name: string,
    public yearsOld: number,
    public grade: string
  ) {}

  getInfo(): string {
    return `Name: ${this.name}, Age: ${this.yearsOld}, Grade: ${this.grade}`;
  }

  isAdult(): boolean {
    return this.yearsOld >= 18;
  }
}
